pyface/data_view/abstract_value_type.py

```diff
@@ -226,6 +235,76 @@ def get_image(self, model, row, column):
         from pyface.image_resource import ImageResource
         return ImageResource("image_not_found")
 
+    def has_check_state(self, model, row, column):
+        """ Whether or not the value has checked state.
+
+        The default implementation returns False.
+
+        Parameters
+        ----------
+        model : AbstractDataModel
+            The data model holding the data.
+        row : sequence of int
+            The row in the data model being queried.
+        column : sequence of int
+            The column in the data model being queried.
+
+        Returns
+        -------
+        has_check_state : bool
+            Whether or not the value has a checked state.
+        """
+        return False
+
+    def get_check_state(self, model, row, column):
+        """ The textual representation of the underlying value.
```

Comment:
I can't find a better docstring but `The textual representation of the underlying value.` doesn't feel appropriate for a `get_check_state`.

Reply:
This is a copy-pasta.  I will fix.